lib/ts/recipe/totp/types.ts

```diff
@@ -0,0 +1,143 @@
+/* Copyright (c) 2023, VRAI Labs and/or its affiliates. All rights reserved.
+ *
+ * This software is licensed under the Apache License, Version 2.0 (the
+ * "License") as published by the Apache Software Foundation.
+ *
+ * You may not use this file except in compliance with the License. You may
+ * obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+import OverrideableBuilder from "supertokens-js-override";
+import { BaseRequest, BaseResponse } from "../../framework";
+import { NormalisedAppinfo } from "../../types";
+import { SessionContainer } from "../session";
+
+export type TypeInput = {
+    issuer: string;
```

Comment:
what should the value of this be from the user's point of view?

Reply:
It represents the app name that will be displayed in the Authy app. Should we just reuse the value from app info?

Reply:
Reusing the value from app info unless user overrides it with the config.